src/core/lib/security/credentials/oauth2/oauth2_credentials.h

```diff
@@ -90,6 +92,10 @@ class grpc_oauth2_token_fetcher_credentials : public grpc_call_credentials {
                             grpc_httpcli_context* httpcli_context,
                             grpc_polling_entity* pollent, grpc_iomgr_cb_func cb,
                             grpc_millis deadline) = 0;
+  virtual void add_additional_metadata(grpc_credentials_mdelem_array*) {
```

Comment:
put the comment outside.
`virtual void add_additional_metadata(grpc_credentials_mdelem_array*) = 0;`

Reply:
Making it a pure virtual means that we need to have implementation on every sub classes. Maybe we want to have default implementation as it has?

Reply:
SG. No need to make it pure.

Reply:
Ok, put the comment above the definition